x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/jinaai/JinaAIService.java

```diff
@@ -386,6 +390,35 @@ public static InferenceServiceConfiguration get() {
                         .build()
                 );
 
+                configurationMap.put(
+                    EMBEDDING_TYPE,
+                    new SettingsConfiguration.Builder(EnumSet.of(TaskType.TEXT_EMBEDDING)).setDescription(
+                        "The type of embedding to return. One of [float, bit, binary]. bit and binary are equivalent and are encoded as "
```

Comment:
How about we construct the [...] dynamically based on the supported values in the enum?